### PR TITLE
OEP-54: Procedures for removing Core Contributors

### PR DESCRIPTION
**Pull request by sarina**
In https://github.com/openedx/open-edx-proposals/pull/275 it was pointed out right at the end of review that there was no caveat for removing Core Contributors. This presents a set of criteria for how that may happen.

### CODE REVIEW

**Comment by sarina** [comment]
Requesting review from any/all who reviewed the initial PR: @felipemontoya @pomegranited @nedbat @davidjoy @pdpinch @antoviaque

File: oeps/processes/oep-0054-core-contributors.rst
Comment (pomegranited) [inline oeps/processes/oep-0054-core-contributors.rst:287]: CCs can approve PRs and merge them without outside approval, so maybe reword this?

```suggestion
* A CC has consistently shown poor judgement, such as merging PRs without due consideration
```

**Review by pomegranited — APPROVED** [review]
Minor recommendation, but the rest LGTM! Thanks for adding this @sarina .

👍 
- [x] I tested this by reading the rendered version.
- [x] I checked for formatting, clarity, and correctness.

File: oeps/processes/oep-0054-core-contributors.rst
Comment (sarina) [reply oeps/processes/oep-0054-core-contributors.rst:287]: Ah, great catch. My intention was something along the lines of "merging PRs without addressing outstanding concerns" - your wording works for me.

**Review by feanil — APPROVED** [review]

File: .gitignore
Comment (nedbat) [inline .gitignore:10]: If you put this in a global gitignore locally, you won't have to add it to individual repos.

Comment (sarina) [reply .gitignore:10]: Ah ok, sounds good, I only use this editor for oeps but i will remove this, sorry

**Review by nedbat — APPROVED** [review]

File: oeps/processes/oep-0054-core-contributors.rst
Comment (antoviaque) [inline oeps/processes/oep-0054-core-contributors.rst:288]: @sarina I'm coming a bit late for the review here it seems -- but what about the cases where core contributors are somewhat active, but don't fulfill the commitments made? With just a vague "activity" requirement, don't we run the risk of ending up with many core contributors who don't do much?

Comment (sarina) [reply oeps/processes/oep-0054-core-contributors.rst:288]: We discussed this at tCRIL and also I did some reading on healthy communities. Having a core contributor who contributes a few patches a year, or only does review, or reviews translations only once per month or quarter... well, that work _is_ valuable. And just because they may be very busy right now does not mean they will be in the future.

We do say below,
> Program administrators will be responsible for determining when a CC is inactive/unreachable, by doing a check at least once per quarter.

I think as part of doing the check we will begin to understand if this concern is warranted or not. As is my prerogative, I tend to not want to overly-solve for situations until we encounter the situation - with context - and figure out where to go from there.

Comment (antoviaque) [reply oeps/processes/oep-0054-core-contributors.rst:288]: @sarina OK, we can see how that goes. But imho we are in a special case of a community, where a lot of the parties deciding on where/how much work is contributed are companies -- not the core contributors themselves. And the goal with this is to help the core contributors getting that time allocated by their bosses, by providing a concrete incentive for the company itself (the claim of having X core contributors). Some companies will play fair regardless, but imho for others if they can get the outcome without having to prioritize it over other company priorities, they won't - and we'll have core contributors saying "we have been busy at our company, so I haven't been able to spend on this".

Comment (antoviaque) [reply oeps/processes/oep-0054-core-contributors.rst:288]: Submitted at https://github.com/openedx/open-edx-proposals/pull/304

**Pull request merged**
